internal/provider/nordvpn/provider.go

```diff
@@ -6,24 +6,25 @@ import (
 
 	"github.com/qdm12/gluetun/internal/constants/providers"
 	"github.com/qdm12/gluetun/internal/provider/common"
-	"github.com/qdm12/gluetun/internal/provider/nordvpn/updater"
+	"github.com/qdm12/gluetun/internal/provider/nordvpn/api"
 	"github.com/qdm12/gluetun/internal/provider/utils"
 )
 
 type Provider struct {
 	storage    common.Storage
 	randSource rand.Source
 	utils.NoPortForwarder
-	common.Fetcher
+	api    *api.API
+	warner common.Warner
 }
 
-func New(storage common.Storage, randSource rand.Source,
-	client *http.Client, updaterWarner common.Warner) *Provider {
+func New(storage common.Storage, randSource rand.Source, client *http.Client, updaterWarner common.Warner) *Provider {
```

Comment:
nit revert this to be on two lines, it's getting pretty long over here (also unneeded change) 😉